src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -155,24 +173,40 @@ private void fireInventoryUpdated() {
             MessageBundle.getMessage("angal.inventory.unitprice.col").toUpperCase(),
             MessageBundle.getMessage("angal.inventory.totalprice.col").toUpperCase()
     };
-    private int[] pColumwidth = { 100, 200, 100, 100, 100, 80, 80, 80 };
-    private boolean[] columnEditable = { false, false, false, false, false, true, true, false };
+    private int[] pColumwidth = { 50, 50, 200, 100, 100, 100, 80, 80, 80 };
+    private boolean[] columnEditable = { false, false, false, false, false, false, true, false, false };
+    private boolean[] columnEditableView = { false, false, false, false, false, false, false, false, false };
+    private boolean[] pColumnVisible = { false, true, true, !GeneralData.AUTOMATICLOT_IN, true, true, true, GeneralData.LOTWITHCOST, GeneralData.LOTWITHCOST };
     private MedicalInventory inventory = null;
     private JRadioButton specificRadio;
     private JRadioButton allRadio;
-    private JTextField searchTextField;
     private JLabel dateInventoryLabel;
     private JTextField codeTextField;
     private String code = null;
-    private String mode = null;
+    private String mode;
+    private JLabel statusLabel;
     private String wardId = "";
     private JLabel referenceLabel;
     private JTextField referenceTextField;
-    private JTextField jTetFieldEditor;
+    private JTextField jTextFieldEditor;
     private JLabel wardLabel;
+    private JLabel chargeLabel;
+    private JLabel dischargeLabel;
+    private JLabel supplierLabel;
+    private JLabel destinationLabel;
     private JComboBox<Ward> wardComboBox;
+    private JComboBox<MovementType> chargeComboBox;
+    private JComboBox<MovementType> dischargeComboBox;
+    private JComboBox<Supplier> supplierComboBox;
+    private JComboBox<Ward> destinationComboBox;
     private Ward wardSelected;
     private JLabel loaderLabel;
+    private boolean selectAll = false;
```

Comment:
```suggestion
    private boolean selectAll;
```
Redundant initialization; the default value is `false`.